deploy/kube-config/influxdb-grafana-controller.yaml

```diff
@@ -19,25 +19,15 @@ desiredState:
               - containerPort: 8083
               - containerPort: 8086
           - name: "grafana"
-            image: "kubernetes/heapster_grafana:v0.5"
+            image: "kubernetes/heapster_influxdb:v0.5"
```

Comment:
Shouldn't we separate out grafana & influxdb into separate pods now for easier deployment? Might want to scale influxdb & grafana separately for example.